nodes/pods/nodes-pods-secrets.adoc

```diff
@@ -64,6 +64,8 @@ include::modules/nodes-pods-secrets-creating-docker.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
 
+include::modules/nodes-pods-secrets-creating-web-console-secrets.adoc[leveloffset=+2]
```

Comment:
There are one typography issue as follow picture. In the GitHub file change, I see that "Additional resources" is before article "Creating a secret using the web console", but in fact the "Additional resources" is on the first paragraph of article.
And refer to other part, I think the "Additional resources" should not put on the first paragraph, it may put before "For more information ...".
![截屏2023-08-04 10 55 52](https://github.com/openshift/openshift-docs/assets/134354288/4f3c3d38-1652-47e9-baae-10ece0931c15)